.ci/pytorch/multigpu-test.sh

```diff
@@ -51,11 +51,9 @@ time python test/run_test.py --verbose -i distributed/tensor/parallel/test_tp_ra
 # FSDP2 tests
 time python test/run_test.py --verbose -i distributed/_composable/fsdp/test_fully_shard_training -- -k test_2d_mlp_with_nd_mesh
 
-# Pipelining composability tests
-time python test/run_test.py --verbose -i distributed/pipelining/test_composability.py
-
 # 3D composability tests
 time python test/run_test.py --verbose -i distributed/_composable/test_composability/test_2d_composable
+time python test/run_test.py --verbose -i distributed/_composable/test_composability/test_pp_composable
```

Comment:
a nit on file naming

it is not really clear which parallelisms should go into 2d and why pp is separate.

for now i think the pp ones are separate mostly because they are using the continuous test base and it doesn't mix nicely with older tests in the same file.

do you have another reason to separate files or is it just becuase of the issue i mentioned?

Reply:
Thank you for the comments. 
That's the only reason for the separating right now.